spec/features/admin_creates_trip_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+feature "Admin features" do
+  describe "admin creates a trip" do
+    context "admin dashboard" do
+      it "has a link to create a trip" do
+        User.create(username: "Alex", password: "password", role: 1)
+        visit root_url
+
+        click_link "Login"
+
+        fill_in("Username", with: "Alex")
+        fill_in("Password", with: "password")
+        click_button("Login")
+
+        click_link "Create a New Trip"
+
+        expect(current_path).to eq(new_admin_trip_path)
+        within('h2') { expect(page).to have_content('Create a New Trip') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.